docs/userguide/antora.yml

```diff
@@ -4,6 +4,8 @@ asciidoc:
     crd: https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions[CRD]
     secret-generation-extension: https://github.com/servicebinding/spec/blob/master/extensions/secret-generation.md[Secret Generation Extension]
     experimental: true
+    servicebinding-title: Service Binding Operator
+    quickstart-namespace: my-postgresql
```

Comment:
if this is used only in quick start guide section, let's define it for now just there: https://docs.antora.org/antora/latest/page/define-and-modify-attributes/#custom-attribute

Reply:
will do, thx